tools/rosconsole/CMakeLists.txt

```diff
@@ -121,3 +121,8 @@ if(CATKIN_ENABLE_TESTING)
     target_link_libraries(${PROJECT_NAME}-thread_test ${PROJECT_NAME})
   endif()
 endif()
+
+catkin_python_setup()
```

Comment:
The package needs a `setup.py` file for this to work. You might have missed to add it to the repo.